brands/infinix/README.md

```diff
@@ -0,0 +1,10 @@
+# Infinix
+
+- Verdict: **⛔ Avoid!**
+
+Infinix is a complete gamble. Some people report that you can use the [standard unlock procedure](/misc/generic-unlock.md), others say you need to use the [Unisoc unlock method][Unisoc Unlock], and some people just say they [aren't unlockable at all.][Infinix lock]
```

Comment:
The post doesn't actually say anything about Infinix not being unlockable. Looks like OP just has an Unislop SOC.

Reply:
turns out google didnt show me a bunch of results when i first did research on infinix and turns out everyone having that issue are just unisoc users -- rewrote the page now that i realized that